css-pseudo-4/Overview.bs

```diff
@@ -603,10 +603,32 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Note: the “paired cascade” behavior of [=highlight colors=]
```

Comment:
We haven't introduced the paired cascading behavior yet, so moving the note that refers to it here is confusing.

Reply:
yeah, it’s a combination of moving this note up from #highlight-cascade (because it’s about default UA styles) and moving the normative text for paired cascade down to #highlight-painting (because it affects used values), which leaves us with a problem here. i’ve tweaked the wording in fa44a161cdba4f8a4f117f53094478576d76ee32, what do you think?